test/readStreamTest.js

```diff
@@ -9,6 +9,70 @@ tape('readStream', function(test) {
     all that painful.
   */
   test.test('readStream should return from all requested types and populate wofRecords', function(t) {
+    function setupTestEnvironment() {
```

Comment:
Not required, but i found the [`temp` library](https://www.npmjs.com/package/temp) that can handle temporary files in a platform agnostic way, and automatically clean up. See [https://github.com/pelias/openaddresses/blob/master/test/parameters.js](https://github.com/pelias/openaddresses/blob/master/test/parameters.js) for usage